lib/presentation/widgets/match_commentary/match_commentary_tile.dart

```diff
@@ -0,0 +1,99 @@
+import 'package:flutter/material.dart';
+import 'package:liga/data/model/live_event_ui_model.dart';
+import 'package:liga/utils/app_colors.dart';
+
+class _Constants {
+  //dimensions
```

Comment:
don't use comments in code, code should be selfexplained. Meanful names for variables